File: api/clients/src/main/java/com/foodagram/clients/users/profile/UsersProfileCreateDto.java
package com.foodagram.clients.users.profile;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UsersProfileCreateDto {
    private String firstName;
    private String lastName;
    private String bio;
    private String city;
    private String country;
    private String website;
    private String jobTitle;
    private Date birthDate;
    private Date createdAt;
    private String instagramProfile;
    private String facebookProfile;
}

File: fooadgram-app/models/user/UsersProfileCreateDto.ts
export interface UsersProfileCreateDto {
    firstName: string;
    lastName: string;
    profilePictureUrl: string;
    bannerPictureUrl: string;
    bio: string;
    city: string;
    country: string;
    website: string;
    jobTitle: string;
    birthDate: Date;
    createdDate: Date;
    instagramProfile: string;
    facebookProfile: string;
}
